common/types.go

```diff
@@ -198,6 +198,8 @@ type Stack struct {
 	Status         string
 	StatusReason   string
 	LastUpdateTime time.Time
+	Region         string
```

Comment:
doesn't seem these are used...is this old?

Reply:
I don't remember, but I did confirm that these attributes (Region & AccountID) were populated but not references anywhere else, so I removed them both.